Assets/AppCenter/Editor/CreateManifest.cs

```diff
@@ -0,0 +1,186 @@
+                .Append(AppCenterSettingsContext.AppCenterPath)
+                .Append("/AppCenter/Plugins/Android/Utility/unarchive.ps1 \"")
+                .Append(" -Source ")
+                .Append(sourceFile)
+                .Append(" -Destination ")
+                .Append(destinationFile)
+                .ToString();
+            processName = "cmd";
+        }
+        else if (Application.platform == RuntimePlatform.OSXEditor)
+        {
+            args = stringBuilder
+                   .Append("-c \"unzip ")
+                   .Append(sourceFile)
+                   .Append(" -d ")
+                   .Append(destinationFile)
+                   .Append(" \"")
+                   .ToString();
+            processName = "/bin/bash";
+        }
+        if (processName.Length > 0) {
+            ExecuteProcess(processName, args);
+        }
+    }
+
+    public static void Create(AppCenterSettings settings)
+    {
+        var loaderZipFile = AppCenterSettingsContext.AppCenterPath + "/AppCenter/Plugins/Android/appcenter-loader-release.aar";
+        var loaderFolder = "appcenter-loader-release";
+        var manifestPath = "appcenter-loader-release/AndroidManifest.xml";
+        var manifestMetafile = "appcenter-loader-release/AndroidManifest.xml.meta";
+
+        if (!File.Exists(loaderZipFile))
+        {
+            UnityEngine.Debug.LogWarning("Failed to load dependency file appcenter-loader-release.aar");
+            return;
+        }
+
+        // Delete unzipeed directory if it already exists.
+        if (Directory.Exists(loaderFolder))
+        {
+            Directory.Delete(loaderFolder, true);
+        }
+
+        UnzipFile(loaderZipFile, loaderFolder);
+        if (!Directory.Exists(loaderFolder))
+        {
+            UnityEngine.Debug.LogWarning("Unzipping loader folder failed.");
+            return;
+        }
+
+        var xmlFile = XDocument.Load(manifestPath);
+
+        var applicationElements = xmlFile.Root.Elements().Where(element => element.Name.LocalName == "application").ToList();
+        if (applicationElements.Count == 0)
+        {
+            UnityEngine.Debug.LogWarning("Incorrect format of AndroidManifest.xml of AppCenterLoader");
+            Directory.Delete(loaderFolder, true);
+            return;
+        }
+        var activityElements = applicationElements[0].Elements().Where(element => element.Name.LocalName == "activity").ToList();
+        
+        // Delete the unzipped folder if the activity element already exists in the AndroidManifest.xml file
+        if (activityElements.Count == 1) 
+        {
+            Directory.Delete(loaderFolder, true);
+            return;
+        }
+
+        var intentelement = new XElement("intent-filter");
+        XNamespace ns = "http://schemas.android.com/apk/res/android";
+        var activityElement = new XElement("activity");
+        activityElement.SetAttributeValue(ns+"name", "com.microsoft.identity.client.BrowserTabActivity");
```

Comment:
```suggestion
        activityElement.SetAttributeValue(ns + "name", "com.microsoft.identity.client.BrowserTabActivity");
```
This line plus a few lines below.